plugins/wordpress-rule-exclusions-before.conf

```diff
@@ -215,15 +218,54 @@ SecRule REQUEST_FILENAME "@endsWith /index.php" \
 SecRule REQUEST_FILENAME "@rx /wp-json/wp/v[0-9]+/(?:posts|pages|users)" \
     "id:9507146,\
     phase:1,\
-    nolog,\
     pass,\
     t:none,\
+    nolog,\
     ver:'wordpress-rule-exclusions-plugin/1.0.1',\
     chain"
     SecRule REQUEST_METHOD "@streq POST" \
         "t:none,\
         setvar:'tx.restricted_headers_basic=/content-encoding/ /proxy/ /lock-token/ /content-range/ /if/ /x-http-method/ /x-method-override/'"
 
+# Loading tags/catagories for pages/posts
+# Obtaining metadata for pages/posts
+# Loading in comments for pages/posts
+SecRule REQUEST_FILENAME "@endsWith /index.php" \
+    "id:9507147,\
+    phase:2,\
+    pass,\
+    t:none,\
+    nolog,\
+    ver:'wordpress-rule-exclusions-plugin/1.0.1',\
+    chain"
+    SecRule ARGS:_fields "@rx ^id[a-z,_]*$" \
```

Comment:
Would `+` instead of `*` make more sense here? Or is there even a minimal number of characters that you know will follow `id`? `_fields=id` looks strange to me.

Reply:
The * is needed so the regex can just match ``id``, _fields will sometimes contain ``id,name`` or similar. I've whitelisted the characters used instead of trying to enumerate all of the possible values.

Reply:
Ok.